felix/bpf-gpl/tc.c

```diff
@@ -625,6 +625,9 @@ static CALI_BPF_INLINE struct fwd calico_tc_skb_accepted(struct cali_tc_ctx *ctx
 	struct cali_tc_state *state = ctx->state;
 	int rc = TC_ACT_UNSPEC;
 	bool fib = true;
+	//this is suppressing a variable set and never used clang warning. Maybe dangerous? 
+	//Is actually used at fwd_fib_set(&fwd, fib) but maybe new clang realizes thats unreachable.
+	CALI_DEBUG("fib=%t", fib);
```

Comment:
This diff should help the build and avoid this debug log.

```diff
diff --git a/felix/bpf-gpl/tc.c b/felix/bpf-gpl/tc.c
index ef7b7d987..a28b8ba5f 100644
--- a/felix/bpf-gpl/tc.c
+++ b/felix/bpf-gpl/tc.c
@@ -1330,6 +1330,7 @@ encap_allow:
        }
 
 deny:
+       (void)fib;
        {
                struct fwd fwd = {
                        .res = TC_ACT_SHOT,
```

Reply:
cc @tomastigera or @sridhartigera to review bpf changes ported to v3.26 release.

Reply:
yep that makes sense pushed